install.sh

```diff
@@ -51,6 +50,37 @@ echo_out() { echo -e "\e[0;37m${*}${RESET}" >&1; } # Use Gray for program output
 # Skip all checks if running on a docker container.
 [[ -f "/.dockerenv" ]] && CREW_FORCE_INSTALL=1
 
+if [[ "$CREW_FORCE_INSTALL" ]]; then
+  echo_info "In container, so skipping hardware checks"
+elif tty | grep -q /dev/pts1; then
+  echo_success "You are in VT-2."
```

Comment:
I could be in VT-3, 4 or 5 too. :)  Maybe we should call it Developer Console.

Reply:
The entire check is only needed for sudo use... so without using sudo, we may not need it at all...